### PR TITLE
Allow log-in by FRN only.

### PR DESCRIPTION
**Pull request by neoeno**
Email is no longer allowed, under the logic that the email might change, but the FRN is constant.

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -41,7 +41,7 @@ en:
         registered_name: Registered name
         percent_total: Percentage total
       user:
-        login: Email or FRN
+        login: Firm Reference Number
```
Comment (munckymagik) [inline config/locales/en.yml]: Why not FRN if that is the accepted abbreviation?

Comment (neoeno) [inline config/locales/en.yml]: If we can assume everyone who has to log-in here has that information, we're fine. But my idea was that if someone doesn't know what an FRN is, they've got no other context on this page that they can use to figure it out. Hence the expansion.

**Comment by munckymagik** [comment]
:+1:

**Comment by darokel** [comment]
:ship:

**Pull request merged**